docs/content.zh/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -359,13 +388,13 @@ data:
     logger.zookeeper.name = org.apache.zookeeper
     logger.zookeeper.level = INFO
 
-    # Log all infos to the console
+    # 将所有 info 级别的日志输出到 console
     appender.console.name = ConsoleAppender
     appender.console.type = CONSOLE
     appender.console.layout.type = PatternLayout
     appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p %-60c %x - %m%n
 
-    # Log all infos in the given rolling file
+    # 将所有 info 级别的日志输出到 rolling file
```

Comment:
here, too.

Reply:
Ok, take your advice. It has been fixed.